src/requests/pin_chat_message.rs

```diff
@@ -0,0 +1,45 @@
+use crate::requests::{ChatId, RequestContext, RequestFuture, ResponseResult, Request};
+use crate::network;
+
+/// Use this method to get up to date information about the chat 
+/// (current name of the user for one-on-one conversations, 
+/// current username of a user, group or channel, etc.). 
+/// Returns a Chat object on success.
+#[derive(Debug, Clone, Serialize)]
+pub struct PinChatMessage<'a> {
+    #[serde(skip_serializing)]
+    ctx: RequestContext<'a>,
+    /// Unique identifier for the target chat or username 
+    /// of the target supergroup or channel (in the format @channelusername)
+    chat_id: ChatId,
+    message_id: i32,
+    disable_notification: bool
```

Comment:
> Optional
```suggestion
    disable_notification: Option<bool>
```

Reply:
It's `false` by default, why make it optional?